Test/plot_error_test.py

```diff
@@ -22,7 +22,7 @@
 >>> ax = cm.plot(plot_lib='seaborn')
 Traceback (most recent call last):
     ...
-pycm.pycm_error.pycmPlotError: Failed to import seaborn module. Please install it using: `pip install seaborn`.
+pycm.pycm_error.pycmPlotError: Failed to import matplotlib module. Please install it using: `pip install matplotlib`.
```

Comment:
This shouldn't be changed, right?
it requested the `seaborn` in line `22`)

Reply:
You are correct, but this issue is related to our code, not the tests.
As both engines use `cm` and `seaborn` is based on `matplotlib`, we check the `matplotlib` installation before checking the `plot_lib`.

https://github.com/sepandhaghighi/pycm/blob/f320c42733b23816947a8639c1998fb2739cd960/pycm/pycm_obj.py#L1064-L1071

Reply:
Oh got it.